bottery/handlers.py

```diff
@@ -18,6 +21,14 @@ def match(self, message):
         raise Exception('Method Not Implemented')
 
 
+class RegexHandler(BaseHandler):
+    def match(self, message):
+        compile = re.compile(self.pattern)
```

Comment:
`compile` is a reserved word :/

Reply:
Would be better if we save the compiled regex to use it later. Something like:

```
if not self.compiled_re:
    self.compiled_re = re.compile(self.pattern)

if compile.match(self.message.text):return True
        return False
```

Reply:
ok!